cookbooks/hdfsdu/README.md

```diff
@@ -0,0 +1,16 @@
+hdfsdu Cookbook
+===============
+Cookbook to build, configure and install HDFS Disk Usage visualization tool.
+
+Usage
+-----
+Pre-reqs: Java, maven and git are installed.
+
+1. Include `recipe[hdfsdu::build.rb]` to build `hdfsdu`.
+2. Include `recipe[hdfsdu::create_user]` in the run-list of all the nodes.
+3. Include `recipe[hdfsdu::fetch_data]` to start oozie coordinator job to fetch fsimage data.
+4. Include `recipe[hdfsdu::deploy]` to start hdfsdu web service.
+
+License and Authors
+-------------------
+Copyright 2016, Bloomberg L.P.
```

Comment:
Oops it is 2017...

Reply:
Updated and added the blurb about APL 2.0 as per [cookbooks/bcpc/NOTICE.txt](https://github.com/bloomberg/chef-bach/blob/59b7667d10c1646136f0ca4e7d600db8bc2824e6/cookbooks/bcpc/NOTICE.txt).